scheduler/pom.xml

```diff
@@ -148,6 +148,30 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- region - Needed for IT testing -->
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-web</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>${javax.servlet-api.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pentaho-platform-repository</artifactId>
+      <version>9.0.0.0-SNAPSHOT</version>
```

Comment:
@ricardosilva88 Hard-coded version on 9.0 Snapshot? Most likely a copy paste error.
You probably want to use `pentaho-repository.version` instead.
https://github.com/pentaho/build-resources/blob/8.2.0.0/version.properties#L114

Reply:
Some how, Intelij add this dependency automatically here when I was creating ScheduleAuthorizationPolicy.java...I am pretty sure that it isn't needed for integration testing. I am preparing to send a new commit, just making some testing to guarantee that it isn't needed at all.